script/prepare_for_import.rb

```diff
@@ -0,0 +1,37 @@
+require "optparse"
+require_relative "../config/environment"
+
+parser = OptionParser.new { |args|
+  args.on "-f", "--fund FUND"
+  args.on "-o", "--organisation ORGANISATION"
+  args.on "-u", "--user EMAIL"
+}
+
+options = {}
+parser.parse!(into: options)
+
+user_email = options.fetch(:user, "roda+dp@dxw.com")
+
+fund = Activity.fund.by_roda_identifier(options.fetch(:fund))
+organisation = Organisation.find_by(name: options.fetch(:organisation))
+user = User.find_by(email: user_email)
+
+unless fund
+  warn "Could not find fund with RODA identifier '#{options.fetch(:fund)}'"
+  exit 1
+end
+
+unless organisation
+  warn "Could not find organisation with name '#{options.fetch(:organisation)}'"
+  exit 1
+end
+
+unless user
+  warn "Could not find organisation with email '#{user_email}'"
+  exit 1
+end
+
+user.organisation = organisation
+user.save
+
+Report.create(fund: fund, organisation: organisation, state: "active")
```

Comment:
It might be worth making sure that this works if invoking the preparation script repeatedly without the full db restore. (But it might not, if the script is only ever intended to be run after a db-restore.)

OTOH, it's the last line, so if it errors because the report already exists, it's not a big deal.